server/src/main/java/com/lunchtime/controllers/EventController.java

```diff
@@ -0,0 +1,33 @@
+package com.lunchtime.controllers;
+
+import com.lunchtime.entity.Event;
+import com.lunchtime.service.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/events")
+public class EventController {
+
+    @Autowired
+    EventService eventService;
+
+    @GetMapping
+    public ResponseEntity<?> getAll() {
+        List<Event> result = eventService.findAll(new Date());
+        return new ResponseEntity<>(result, HttpStatus.OK);
+    }
+
+    @PostMapping
+    public ResponseEntity<?> addEvent(@RequestBody @Valid Event event) {
```

Comment:
Should't be the anotation  @Valid for @RequestBody

Reply:
I do not agree. When I delete annotation and add a new object with wrong parameters, i get server error.

Reply:
I mean, you should use in this case (@Valid @RequestBody Event eventt). Also @Valid must stay for @RequestBody

Reply:
@Andrii35 could you explain, why must this annotation order be exactly this?

Reply:
I read many examles about this annotations. It is nothing is told what the difference is where the annotation stands. But in only one case did I see the location as in Roman. For better readability of the code I would put @Valid for @RequestBody.